### DISCUSSION TITLE
connected but not syncing

### DISCUSSION THREAD

**Discussion by antonello22** [Chia-Network/chia-blockchain#6576] [Q&A]
ya guys i start to be fed up,
there are updates after updates, 
now even new updates are worst than the previous....
with this one it looks like i will loose the plots they were in progress!!!!
Where is the debug team before releasing the update? 
Maybe do less update and take your time to release a proper one!!!!
this is not a good path for success .....

**Comment by antonello22** [comment]
so this one i am connected but not syncing
port 8444 is open
anyone has the same problem?

**Reply by lookWhatIvanDid** [reply]
I have NO SYNC problem always. I've never seen it done yet. I mean it stops when I start making a plot, and never gets fully sync when I don't. :(
